src/components/ControlPanel/reducers/controlPanelSectionsReducer.test.js

```diff
@@ -0,0 +1,153 @@
+import {
+  INITIAL_STATE,
+  CONTROL_PANEL_SECTIONS_ACTIONS,
+  controlPanelSectionsReducer,
+} from './controlPanelSectionsReducer';
+
+describe('controlPanelSectionsReducer', () => {
+  it("should fetch control panel's sections", () => {
```

Comment:
Podrías usar una palabra diferente de fetch? (en realidad el fetch ocurre fuera del reducer), algo como `it should update control panel's sections on fetch` (no lo pensé mucho)

Reply:
Es verdad...